### DISCUSSION TITLE
Support for project collaboration

### DISCUSSION THREAD

**Discussion by nanashili** [AuroraEditor/AuroraEditor#684] [Ideas]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

The feature request would add support for allowing users to share their projects with friends/co-workers virtually. By getting on a call they can work on the project together each being able to edit on the project.

I suggest we use https://livekit.io

### Describe alternatives you've considered

None

### Additional context

None

**Comment by aurora-care-bear** [comment]
Thanks for submitting a issue!
to assign this issue to yourself please comment `@aurora-care-bear please assign me`

**Comment by Maartz** [comment]
I was reading the docs.
It seems it has to spawn up a server.
IDK what are the limitations within an app when it comes to expose some port etc.
I'd reckon it is trivial since AE needs to expose a port as well for the updates, correct?

**Comment by 0xWDG** [comment]
> I was reading the docs. It seems it has to spawn up a server. IDK what are the limitations within an app when it comes to expose some port etc. I'd reckon it is trivial since AE needs to expose a port as well for the updates, correct?

I think the safest option is to connect to the AE API and let handle that as proxy/server so that there will be no direct connection between clients (and open ports).
Maybe at some point support local -in network- collab
